client/src/components/TaskFilterSelector/TaskFilterSelector.module.css

```diff
@@ -0,0 +1,25 @@
+.selectedFilter {
+  margin: 5px 10px;
+  border-color: #8498b2;
+  border-width: 2px;
+  border-radius: 10px;
+  border-style: solid;
+  color: white;
+  background: #032a5c;
+  width: 15%;
+}
+
+.filterButton {
```

Comment:
I would also add in a hover style (to show the pointer) when a filter is hovered

Reply:
Noted. Will put this on hold, prioritizing functionality.